controllers/teams/poster.js

```diff
@@ -0,0 +1,49 @@
+const { team } = require('../../databases/db');
+const { imgController, fileController } = require('../filesControllers'); 
+
+const allowedFields = ['name', 'badge', 'idEvents'];
+
+/**
+ * It creates a new team in the database, and returns the newly created team.
+ * @param req - the request object
+ * @param res - the response object
+ */
+
+const post = async (req, res) => {
+  try {
+    const { name, idEvents: eventsid } = req.body;
+    const img_relative_dir = '/' + imgController.img_relative_dir.replaceAll('\\', '/') + '/';
+    let filepath;
+    if (process.env.USINGIMGHOST == 'true') {
+      filepath = `${process.env.IMGURL}${img_relative_dir}${req.file.filename}`;
+    } else {
+      filepath = `${process.env.OFFSIDEURL}${img_relative_dir}${req.file.filename}`;
+    }
```

Comment:
Este approach esta dudosisimo xD, pero................. independientemente de.... haz que esto sea un helper por que se va a usar en demaciados sitios

```suggestion
    filepath = getImagePath(req.file.filename);
```

En `/src/helpers/get-image-path.js`:

```
function getImagePath(image) {
   const img_relative_dir = '/' + imgController.img_relative_dir.replaceAll('\\', '/') + '/';

    if (process.env.USINGIMGHOST == 'true') {
      return `${process.env.IMGURL}${img_relative_dir}${image}`;
    }
    
    return`${process.env.OFFSIDEURL}${img_relative_dir}${image}`;
}

```

Reply:
Ok papi

Reply:
Luego con mas tiempo vemos la otra correcion